packages/eslint-plugin/docs/rules/no-confusing-void-expression.mdx

```diff
@@ -119,6 +119,24 @@ function doSomething() {
 console.log(void alert('Hello, world!'));
 ```
 
+### `ignoreVoidReturningFunctions`
+
+Whether to ignore returns from functions with explicit `void` return types and functions with contextual `void` return types.
+
+It may be preferable to allow functions that explicitly return `void` to return `void` expressions. This exception will also apply to functions with an implicit contextual return type of `void`.
```

Comment:
[Docs] It's good to be explicit on _why_ something is or isn't preferable:

```suggestion
Some projects prefer allowing functions that explicitly return `void` to return `void` expressions. Doing so allows more writing more succinct functions.

:::note
This is technically risky as the `void`-returning function might actually be returning a value not seen by the type system.
:::
```

Reply:
Can't agree more on this, thanks 👍